src/components/home/components/Home.js

```diff
@@ -1,40 +1,21 @@
 import React from 'react';
-import { Button, Card, Typography, Row, Col } from 'antd';
-import { Link } from 'react-router-dom';
 import layouts from '../../layouts';
-
-import { SIGNIN, SIGNUP, TITLE_TEXT, CARD_SUBTITLE } from '../constants';
+import art from '../../../images/illustration.png';
+import text from '../../../images/text.png';
```

Comment:
host these images in firestore, then below point the `src` at the firebase url for the given image

Reply:
Alright but those are not real images, just a placeholder